packages/atomic/src/components/search/facets/atomic-automatic-facet-builder/atomic-automatic-facet-builder.tsx

```diff
@@ -0,0 +1,96 @@
+import {BooleanValue, NumberValue, Schema} from '@coveo/bueno';
+import {
+  AutomaticFacetBuilder,
+  AutomaticFacetBuilderState,
+  SearchStatus,
+  buildAutomaticFacetBuilder,
+  buildSearchStatus,
+} from '@coveo/headless';
+import {Component, Prop, State, h} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+
+/**
+ * @beta - This component is part of the automatic facets feature.
```

Comment:
`@beta` should this be `@internal` ?

Reply:
Not sure if it is customary to mention "Automatic facets are currently in beta testing and should be available soon." - guess it is implied when marked

Reply:
@beta was the tag Oli told me to use. I already used it in other places in the code that is not present in this pr. I think it is fine as it is but I asked Sam to review the strings. @srosenberg-coveo